app/policies/resource_policy.rb

```diff
@@ -9,4 +9,8 @@ def initialize(user, resource)
   def show?
     true
   end
+
+  def update?
+    true
```

Comment:
Yeah, for the time being this is fine. We don't really have a way for people to edit resources on the site now anyway.